app/views/unattended/provisioning_templates/registration/global_registration.erb

```diff
@@ -99,7 +99,7 @@ echo "# Running registration"
 echo "#"
 
 <% if plugin_present?('katello') -%>
-if [ x$ID = xrhel ] || [ x$ID = xcentos ] || [ x$ID = xol ]; then
+if [ "${ID_LIKE#*fedora*}" != "$ID_LIKE" ]; then
```

Comment:
```suggestion
if [ -f /etc/redhat-release ]; then
```

Reply:
But this would enable registration with `subscription-manager` for Fedora, I'm not sure if `subscription-manager` is available for Fedora.

Reply:
It certainly is:
```console
# dnf info subscription-manager
Last metadata expiration check: 1:24:14 ago on Wed 15 Sep 2021 11:51:57 CEST.
Available Packages
Name         : subscription-manager
Version      : 1.29.18
Release      : 1.fc34
Architecture : x86_64
Size         : 883 k
Source       : subscription-manager-1.29.18-1.fc34.src.rpm
Repository   : updates
Summary      : Tools and libraries for subscription and repository management
URL          : http://www.candlepinproject.org/
License      : GPLv2
Description  : The Subscription Manager package provides programs and libraries to allow users
             : to manage subscriptions and yum repositories from the Red Hat entitlement
             : platform.
```

Reply:
Suggestion from @ekohl  seems reasonable to me, I have updated the commit.